docs/framework/configure-apps/file-schema/compiler/compiler-element.md

```diff
@@ -108,5 +108,4 @@ Specifies the compiler configuration attributes for a language provider.
  <xref:System.CodeDom.Compiler.CodeDomProvider>  
  [Configuration File Schema](../../../../../docs/framework/configure-apps/file-schema/index.md)  
  [\<compilers> Element](../../../../../docs/framework/configure-apps/file-schema/compiler/compilers-element.md)  
- [Specifying Fully Qualified Type Names](../../../../../docs/framework/reflection-and-codedom/specifying-fully-qualified-type-names.md)  
- [compiler Element for compilers for compilation (ASP.NET Settings Schema)](http://msdn.microsoft.com/library/f7d6b078-5d42-4134-b3f7-62e1aba1df1e)
```

Comment:
do we want to completely remove this or change to a version that works?
https://msdn.microsoft.com/en-us/library/f7d6b078-5d42-4134-b3f7-62e1aba1df1e(v=vs.100)

Reply:
Lets err on the side of maintainability and add the version that works.